### ISSUE TITLE
sbt 0.10.1 version has incorrect pom file

### ISSUE THREAD

**Issue by teamon** [softprops/coffeescripted-sbt#2]
http://repo.lessis.me/me/lessis/coffeescripted-sbt_2.8.1/0.1.2-0.10.1/coffeescripted-sbt_2.8.1-0.1.2-0.10.1.pom 

has dependency for sbt 0.10.0 (which makes conflict when use from sbt 0.10.1)

**Comment by softprops** [comment]
Hrm, that should have been killed with this commit [1](https://github.com/softprops/coffeescripted-sbt/commit/4fd82e0967558eec943351bd201f406560739799) before I did the release of 0.1.2. Previously it was recommended to drop a build.properties file in the project directory [2](https://github.com/softprops/coffeescripted-sbt/tree/0.1.2/project) to specify the sbt version. Since I removed it. I'm not sure why there's an explicit dependency. I'll have another look tonight. Thanks for reporting this.
